### PR TITLE
fix(DTFS-6464): use summary lists on underwriting page

### PR DESCRIPTION
**Pull request by MarRobSoftwire**
## Introduction
On the underwriting page the exporter table does not have the headers semantically marked up

## Resolution
* Changed the exporter and facilities tables to use `govukSummaryList` components

### CODE REVIEW

File: trade-finance-manager-ui/templates/case/underwriting/pricing-and-risk/_macros/section-exporter/exporter-table.njk [hunk 1]
```diff
@@ -1,100 +1,162 @@
...
-
-            {% else %}
-
-            {{ govukTag({
-              text: 'Not added',
-              classes: 'govuk-tag--yellow',
+  
+    {% set exporterCreditRatingHeadingHtml %}
+      <span data-cy="exporter-table-credit-rating-heading">Credit rating</span>
+    {% endset %}
```
Comment (lmarrai-sw) [inline trade-finance-manager-ui/templates/case/underwriting/pricing-and-risk/_macros/section-exporter/exporter-table.njk:13]:
```suggestion
  {% set exporterCreditRatingHeadingHtml %}
    <span data-cy="exporter-table-credit-rating-heading">Credit rating</span>
  {% endset %}
```

I think the indenting is 1 level too much here

File: trade-finance-manager-ui/templates/case/underwriting/pricing-and-risk/_macros/section-exporter/exporter-table.njk [hunk 2]
```diff
@@ -1,100 +1,162 @@
...
+        }
+      } %}
+    {% else %}
+      {% set creditRatingRow = {
+        key: { html: exporterCreditRatingHeadingHtml },
+        value: { html: exporterCreditRatingValueHtml }
+      } %}
+    {% endif %}
+
+  {% endif %}
```
Comment (lmarrai-sw) [inline trade-finance-manager-ui/templates/case/underwriting/pricing-and-risk/_macros/section-exporter/exporter-table.njk:79]: I had some trouble reading these nested ifs. I think separating the nesting like below would make it more readable. The reason I think this is more readable (other than less nesting generally being easier to read) is that it's (hopefully) more apparent that `exporterCreditRating` determines what text we show and `userCanEditGeneral` determines whether or not we show an action, whereas I had to work harder to understand that from the nested ifs


```
  {% if exporterCreditRating %}
    {% set exporterCreditRatingValueHtml %}
      <span data-cy="exporter-table-credit-rating-value">{{ exporterCreditRating }}</span>
    {% endset %}
    {% set editExporterCreditRatingActionText = 'Change' %}
    {% set editExporterCreditRatingActionDataCyValue = 'exporter-table-change-credit-rating-link' %}
  {% else %}
    {% set exporterCreditRatingValueHtml %}
      {{ govukTag({
        text: 'Not added',
        classes: 'govuk-tag--yellow',
        attributes: {
          'data-cy': 'exporter-table-credit-rating-not-added'
        }
      }) }}
    {% endset %}
    {% set editExporterCreditRatingActionText = 'Add' %}
    {% set editExporterCreditRatingActionDataCyValue = 'add-credit-rating-link' %}
  {% endif %}
  
  {% if userCanEditGeneral %}
    {% set creditRatingRow = {
      key: { html: exporterCreditRatingHeadingHtml },
      value: { html: exporterCreditRatingValueHtml },
      actions: {
        items: [
          {
            href: "/case/" + caseId + "/underwriting/pricing-and-risk/edit",
            text: editExporterCreditRatingActionText,
            attributes: {
              'data-cy': editExporterCreditRatingActionDataCyValue
            }
          }
        ]
      }
    } %}  
  {% else %]
    {% set creditRatingRow = {
      key: { html: exporterCreditRatingHeadingHtml },
      value: { html: exporterCreditRatingValueHtml }
    } %}
  {% endif %}
```

**Review by lmarrai-sw — COMMENTED** [review]
This looks good thanks @MarRobSoftwire

File: trade-finance-manager-ui/templates/case/underwriting/pricing-and-risk/_macros/section-exporter/exporter-table.njk [hunk 3]
```diff
@@ -1,100 +1,137 @@
...
+
+  {% if userCanEditGeneral %}
+    {% set probabilityOfDefaultRow = {
+      key: { html: probabilityOfDefaultHeadingHtml },
+      value: { html: probabilityOfDefaultValueHtml },
+      actions: {
+        items: [
+          {
+            href: "/case/" + caseId + "/underwriting/pricing-and-risk/probability-of-default",
+            text: "Change",
```
Comment (avaitonis) [inline trade-finance-manager-ui/templates/case/underwriting/pricing-and-risk/_macros/section-exporter/exporter-table.njk:115]: `Change` context is clear for people with sight, but for screen readers we should add `visuallyHiddenText` or `<span class="govuk-visually-hidden">`. 
 
 Requirement example - https://ukef-dtfs.atlassian.net/browse/DTFS2-6467

File: trade-finance-manager-ui/templates/case/underwriting/pricing-and-risk/_macros/section-exporter/exporter-table.njk [hunk 4]
```diff
@@ -1,100 +1,137 @@
...
+
+  {% if userCanEditGeneral %}
+    {% set lossGivenDefaultRow = {
+      key: { html: lossGivenDefaultHeadingHtml },
+      value: { html: lossGivenDefaultValueHtml },
+      actions: {
+        items: [
+          {
+            href: "/case/" + caseId + "/underwriting/pricing-and-risk/loss-given-default",
+            text: "Change",
```
Comment (avaitonis) [inline trade-finance-manager-ui/templates/case/underwriting/pricing-and-risk/_macros/section-exporter/exporter-table.njk:78]: `Change` context is clear for people with sight, but for screen readers we should add `visuallyHiddenText` or `<span class="govuk-visually-hidden">`.

Requirement example - https://ukef-dtfs.atlassian.net/browse/DTFS2-6467

File: trade-finance-manager-ui/templates/case/underwriting/pricing-and-risk/_macros/section-exporter/exporter-table.njk [hunk 5]
```diff
@@ -1,100 +1,137 @@
...
+
+  {% if userCanEditGeneral %}
+    {% set creditRatingRow = {
+      key: { html: exporterCreditRatingHeadingHtml },
+      value: { html: exporterCreditRatingValueHtml },
+      actions: {
+        items: [
+          {
+            href: "/case/" + caseId + "/underwriting/pricing-and-risk/edit",
+            text: editExporterCreditRatingActionText,
```
Comment (avaitonis) [inline trade-finance-manager-ui/templates/case/underwriting/pricing-and-risk/_macros/section-exporter/exporter-table.njk:41]: `Add/Change` context is clear for people with sight, but for screen readers we should add `visuallyHiddenText` or `<span class="govuk-visually-hidden">`.

Requirement example - https://ukef-dtfs.atlassian.net/browse/DTFS2-6467

**Review by avaitonis — DISMISSED** [review]
LGTM

**Pull request merged**